plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -2377,7 +2378,7 @@ public Type getType() {
     }
 
     private Map<String, String> getVersionStrings() {
-        final Script command = new Script(_versionstringpath, _timeout, s_logger);
+        final Script command = new Script(_versionstringpath, _timeout.getMillis(), s_logger);
```

Comment:
Please consider adding an overridden version of the `Script(String, int, Logger)` constructor that accepts `Duration` to encapsulate this type conversion.